Formula/hyperkit.rb

```diff
@@ -0,0 +1,120 @@
+class Hyperkit < Formula
+  desc "Lightweight virtualization hypervisor for MacOS"
+  homepage "https://github.com/moby/hyperkit"
+  url "https://github.com/moby/hyperkit.git",
+    :tag => "v0.20170425",
+    :revision => "a9c368bed6003bee11d2cf646ed1dcf3d350ec8c"
+
+  bottle do
+    root_url "http://dl.bintray.com/markeissler/homebrew/bottles"
+    cellar :any_skip_relocation
+    sha256 "5cfa72e41bad9d812206a9850d7e6e63185ce1bffa0d1718beb5f09734d9bb29" => :sierra
+    sha256 "32162cf81ca23a27f97e0fe0727ecc6f3dcf179716ac77fb9fbc38883e3d114f" => :el_capitan
+    sha256 "0fb4cf0f9f8d81eb1be99d620b454ab14ec8680237f6938db095b032874e821d" => :yosemite
+  end
+
+  head do
+    url "https://github.com/moby/hyperkit.git", :branch => "master"
+  end
+
+  depends_on "opam"
+  depends_on "libev"
+
+  resource "tinycorelinux" do
+    url "https://dl.bintray.com/markeissler/homebrew/hyperkit-kernel/tinycorelinux_8.x.tar.gz"
+    sha256 "560c1d2d3a0f12f9b1200eec57ca5c1d107cf4823d3880e09505fcd9cd39141a"
+  end
+
+  def install
+    ohai "Installing hyperkit dependencies with OPAM. This might take a while..."
```

Comment:
This part of the install can take a long time during which the user might think the process is hung. I'm not entirely clear what the point is of removing a helpful message. As mentioned previously, it's in the guidelines.